class.jetpack.php

```diff
@@ -1232,6 +1232,17 @@ public static function file_system_write_access() {
 		return 0;
 	}
 
+	public static function is_vcs_dir( $check_dir ) {
+		$checkout = false;
+		$check_dir = untrailingslashit( $check_dir );
+		$vcs_dirs = array( '.svn', '.git', '.hg', '.bzr' );
+		foreach ( $vcs_dirs as $vcs_dir ) {
+			if ( $checkout = @is_dir( rtrim( $check_dir, '\\/' ) . "/$vcs_dir" ) )
```

Comment:
@lezama 
is there a case where it's necessary to `untrailingslashit` and then again `rtrim` within the loop? 
If there's a case where the `rtrim` it's necessary, maybe it could be done once outside of the loop?
`if` block is missing curly brackets